cfgov/templates/find_a_housing_counselor.html

```diff
@@ -5,6 +5,12 @@
 Find a housing counselor
 {% endblock %}
 
+{% block analytics_js %}
+<!-- Google Optimize --> 
```

Comment:
Well, there aren't any references to `_gaq` in the `find-a-housing-counselor` template/code, but I just found some in one of the js files the non-responsive template loads. It's probably safest to just call super now in case there's more analytics code lurking in other dependencies.   

Fixed in https://github.com/cfpb/cfgov-refresh/pull/2370/commits/896ee8d9e2c8752d2ccf942d29b34c9abd3a756c